apps/web/layouts/checkout.vue

```diff
@@ -32,22 +32,49 @@ import { SfIconArrowBack, SfLoaderCircular } from '@storefront-ui/vue';
 import type { CheckoutLayoutProps } from './types';
 
 const localePath = useLocalePath();
-const { t } = useI18n();
+const i18n = useI18n();
+const { t } = i18n;
 const router = useRouter();
 const { isAuthorized } = useCustomer();
 const { data: cart, loading: isLoading } = useCart();
 const { setInitialData } = useInitialSetup();
 const viewport = useViewport();
 const { heading, backLabelMobile, backLabelDesktop } = defineProps<CheckoutLayoutProps>();
+
+const currentLocale: string =
```

Comment:
this is all a bit too much for a layout component. 

I also dont like the destructing of the path and all the additional logic.
We need to find a better solution that uses more of i18n https://i18n.nuxtjs.org/docs/api/nuxt methods. 

maybe we should set a backlink var object with a link and a lang { url, lang } and if we click on the button and if { locale} =  usei18n is not the same then we redirect to `paths.home`